lib/types/api.types.ts

```diff
@@ -238,21 +246,30 @@ export interface BaseAppSDK {
   cmaAdapter: Adapter
 }
 
-export type EditorAppSDK = Omit<BaseAppSDK, 'ids'> &
+export type EditorAppSDK<
+  TInstallationParameters extends KeyValueMap = KeyValueMap,
+  TInstanceParameters extends KeyValueMap = KeyValueMap
+> = Omit<BaseAppSDK<TInstallationParameters, TInstanceParameters, never>, 'ids'> &
```

Comment:
I wish there was a way to bubble up the generics from BaseAppSDK. But this looks like a good solution!